01-Setup and getting started/README.md

```diff
@@ -292,6 +292,9 @@ Setup will create infra resources and schedule daily security control scan on ta
 > 1. _Setup may take up to 5 minutes to complete._
 > 2. _For better performance, we recommend using one location for hosting central scanning user-assigned MI and resources which will be created in the following installation steps using the `Install-AzSKTenantSecuritySolution` cmdlet._
 > 3. _To restrict network traffic and to ensure that all inbound communication to critical backend resources of AzTS solution are routed through private network(VNet), install AzTS setup with **VNet integration**. For this you will need to run the installation command `Install-AzSKTenantSecuritySolution` with `-EnableVnetIntegration` switch._
+>
+>     **Note:** Enabling VNet integration for AzTS setup is not recommended as of now due to ongoing issue for connectivity to Application Insights over Private link scope
```

Comment:
Can we link the 'ongoing issue' to the GitHub issue?

Reply:
Added link for 'ongoing issue' to the GitHub issue